### ISSUE TITLE
Accessibility: tab navigation changed since WordPress 5.2

### ISSUE THREAD

**Issue by jeherve** [WordPress/gutenberg#17847]
When using WordPress' current stable release, you can rely on the `Tab` key to quickly start composing:
1. Focus is set to the title as you open the editor.
2. Once you've added a title, you can hit `Tab` to quickly get to the main content area, where you'll be able to add blocks (by hitting `/` for example).

This changed with the current version of Gutenberg.

1. Go to Posts > Add New
2. Enter a post title
3. Hit Tab.
4. Start typing. It turns out you are not in the main content area.

**Desktop (please complete the following information):**
 - OS: macOS Catalina
 - Browser: Chrome
 - Version: 77.0.3865.90

**Comment by brentswisher** [comment]
Tested and verified. It appears to be a side effect of #16500 similar to #17776. 

@youknowriad - any thoughts on this? If I disable navigation mode in WritingFlow it prevents this, but I am not sure where/how that should be implemented. I could pull isCleanNewPost from core/editor, but I'm thinking I shouldn't do that in this package? 

Code that fixes the issue using isCleanNewPost as proof of concept here: https://github.com/WordPress/gutenberg/commit/5cdc3d247b08f02a346a6ba055ecf9336c7d8165

**Comment by youknowriad** [comment]
This was intentional, the idea is that by default you're in "Navigation Mode" to ease cross-block navigation for screen reader and keyboard users. 

You can hit "Enter" to enter "edit mode". 

We're considering explicit separation between these two modes here #17088 and potentially changing the default mode.

**Comment by brentswisher** [comment]
@youknowriad I think #17088 could help alleviate this, but I would question whether starting a new post in navigation mode eases navigation **specifically on a new post**. If you compare the two processes:

## "Old" Process:
1. Click add New Post
2. Title is auto-focused
3. Add a title, hit tab
4. The cursor is inserted into the first paragraph and you can begin composing

**Screenshot:**
![H6P7JZ9X53](https://user-images.githubusercontent.com/6653970/66847987-4b32a100-ef42-11e9-9561-a316f37a8489.gif)

## "New" Process:
1. Click add New Post
2. Title is auto-focused
3. Add a title, hit tab
4. The cursor is moved to the first paragraph in navigation mode
5. If I start typing text, nothing happens. There is no indication of why what I am typing is not on the screen or what to do to make that happen.
6. If I hit enter, I am dropped into "edit" mode and can enter text (But it is not clear that this is what I need to do)

**Screenshot:**
![spyQx38ret](https://user-images.githubusercontent.com/6653970/66847911-2b02e200-ef42-11e9-89dc-57a02754a3f0.gif)

The fact that as I type when I first tab to the paragraph there is no indication to me what is happening seems problematic. With no other content to actually navigate to, I'm questioning the logic of starting in navigation mode? Maybe it would be better to only default it to navigation when editing an existing post?

**Comment by youknowriad** [comment]
I agree, it's not perfect, and yeah, this mode have been in the plugin for just a month or two so we can iterate on it and adapt it. I wouldn't mind a PR to switch the default (and check with a11y)

**Comment by gziolo** [comment]
Should we raise for the next accessibility weekly meeting? It would be great to decide whether it’s fine as is or this should be offered as an option activated upon using esc key. I personally find it confusing in some cases as it isn’t predictable when mixing the mouse/trackpad and keyboard usage.
